src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -0,0 +1,220 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include "ie_parallel.hpp"
```

Comment:
parallelization seems to be not used inside the impl

Reply:
remove in 0a858793244bc2da0c635de34de8cb9248a9f67b